.github/workflows/continuous_integration.yml

```diff
@@ -234,18 +243,28 @@ jobs:
     - name: UM global against known good output (KGO)
       run: |
         cd driver
-        KGO=data/outputs/UKMO/cosp2_output.um_global.gfortran.kgo.nc
+        KGO=data/outputs/UKMO/cosp2_output.um_global.gfortran.kgo.$KGO_VERSION.nc
         TST=data/outputs/UKMO/cosp2_output.um_global.nc
         if [[ -e /opt/intel/oneapi/setvars.sh ]]; then
           ATOL=1.0e-20
           RTOL=0.0006
           OUTTST=data/outputs/UKMO/cosp2_output.um_global.out
-          OUTKGO=data/outputs/UKMO/cosp2_output.um_global.ifort.kgo.out
+          OUTKGO=data/outputs/UKMO/cosp2_output.um_global.ifort.kgo.$KGO_VERSION.out
           python compare_to_kgo.py ${KGO} ${TST} --atol=${ATOL} --rtol=${RTOL} \
               --noerror=True --stats_file=${OUTTST}
           diff ${OUTKGO} ${OUTTST}
         else
-          python compare_to_kgo.py ${KGO} ${TST} --atol=${ATOL} --rtol=${RTOL}
+          if [[ "${F90}" = 'gfortran' ]]; then
+            python compare_to_kgo.py ${KGO} ${TST} --atol=${ATOL} --rtol=${RTOL}
+          else
+            ATOL=1.0e-20
+            RTOL=0.0006
+            OUTTST=data/outputs/UKMO/cosp2_output.um_global.out
+            OUTKGO=data/outputs/UKMO/cosp2_output.um_global.gfortran.kgo.$KGO_VERSION.out
+            python compare_to_kgo.py ${KGO} ${TST} --atol=${ATOL} --rtol=${RTOL} \
```

Comment:
Rather than repeat the line, maybe move it outside the IF block?

Reply:
Yes, it was messy.  I'm now using different KGOs for each version, making it easier to adapt in the future.